### PR TITLE
Splitter Demo: create Splitter demo

### PR DESCRIPTION
**Pull request by nikkithelegendarypokemonster**

### CODE REVIEW

File: MVCDemos/Content/DemosStyles/Splitter/Overview.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    border: 1px solid var(--dx-color-border);
+    border-radius: 8px;
+    overflow: hidden;
+  }
+  
+  .pane-content {
+    padding: 12px;
+  }
+  
+  .pane-title {
```
Comment (EugeniyKiyashko) [inline MVCDemos/Content/DemosStyles/Splitter/Overview.css]: Indentions are incorrect

File: MVCDemos/Views/Splitter/Overview.cshtml [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+<script>
+    function GetStateText(data) {
+        if (data.resizable !== false && !data.collapsible) {
+            return 'Resizable only';
+        }
+        return `${data.resizable ? 'Resizable' : 'Non-resizable'} and ${data.collapsible ? 'collapsible' : 'non-collapsible'}`;
+    }
+
+    function paneContentTemplate(data, paneName) {
+        var content = "<div class='pane-content'>" +
```
Comment (EugeniyKiyashko) [inline MVCDemos/Views/Splitter/Overview.cshtml]: Use string literals instead of concatenation

File: MVCDemos/Views/Splitter/Overview.cshtml [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+                .Template((data) => {
+                    return "<script>paneContentTemplate(data, 'Right Pane');</script>";
+                });
+            })
+        )
+    </div>
+</div>
+
+<script>
+    function GetStateText(data) {
```
Comment (EugeniyKiyashko) [inline MVCDemos/Views/Splitter/Overview.cshtml]: We shouldn't use uppercase for the first letter

File: NetCoreDemos/Views/Splitter/Overview.cshtml [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+    function paneContentTemplate(data, paneName) {
+        const dimensionOptions = new Set(['size', 'minSize', 'maxSize']);
+        const filteredData = Object.fromEntries(
+            Object.entries(data)
+                .filter(([key]) => dimensionOptions.has(key))
+        );
+
+        let content = `
+        <div class='pane-content'>
```
Comment (EugeniyKiyashko) [inline NetCoreDemos/Views/Splitter/Overview.cshtml]:
```
return `
    <div class='pane-content'>
        <div class='pane-title'>
            ${paneName}
        </div>
        <div class='pane-state'>
            ${getStateText(data)}
        </div>
        ${Object.entries(filteredData).map(([key, value]) => `
            <div class="pane-option">${key}: ${value}</div>
        `).join('')}
    </div>
`;
```

**Review by AlexanderMoiseev — DISMISSED** [review]

File: NetCoreDemos/Views/Splitter/Overview.cshtml [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+                                        .Template(new JS("function(data){return paneContentTemplate(data, 'Nested Right Pane')}"));
+                                });
+                            });
+                    });
+                });
+
+                items.Add()
+                    .Resizable(false)
+                    .Size("140px")
+                    .Collapsible(false)
```
Comment (mpreyskurantov) [inline NetCoreDemos/Views/Splitter/Overview.cshtml:50]: May be put Collapsible after Resizable like you do it everywhere here...
However, ever without this, it looks good to me

**Review by mpreyskurantov — DISMISSED** [review]

**Review by AlexanderMoiseev — DISMISSED** [review]

**Review by AlexanderMoiseev — APPROVED** [review]

**Review by EugeniyKiyashko — APPROVED** [review]

**Pull request merged**